### DISCUSSION TITLE
Welcome to kitty Discussions!

### DISCUSSION THREAD

**Discussion by kovidgoyal** [kovidgoyal/kitty#3156] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sheriferson** [comment]
Nice to see this pop up :) I've been using kitty since December 2017. I wish I remember how I learned about it, but I'm a huge fan. I spend most of my working – work work and personal project work – time in kitty and its features make command-line life very pleasant.

Since I want the tools I love to stick around and kitty (and all the research and knowledge that were needed for it) is hard work, I support @kovidgoyal on Patreon and I recommend other heavy users to do so as well. Also thanks to @Luflosi for contributing so much.

Oh fwiw I use a [light color scheme](https://github.com/sheriferson/dotfiles/blob/master/kitty/light.conf), and here's my [custom configuration](https://github.com/sheriferson/dotfiles/blob/master/kitty/custom.conf). Curious to see others' as well.

@kovidgoyal I wonder, since this is your creation, do you use kitty with all the default configurations set? If not, I'm curious what you change.

**Comment by kovidgoyal** [comment]
It's mostly just font and few shortcuts, here's my kitty.conf
```
font_family Operator Mono Book
bold_font Operator Mono Medium
italic_font Operator Mono Book Italic
bold_italic_font Operator Mono Medium Italic
remember_window_size no

symbol_map U+E0A0-U+E0A2,U+E0B0-U+E0B3 PowerlineSymbols
# Rule for / which is much wider in the italic Operator Mono variant, leading to truncated rendering
symbol_map U+002f Operator Mono Book 
map kitty_mod+f4 send_text all ρσ
# Rule for Nerd Fonts
symbol_map U+23FB-U+23FE,U+2B58,U+E200-U+E2A9,U+E0A0-U+E0A3,U+E0B0-U+E0BF,U+E0C0-U+E0C8,U+E0CC-U+E0CF,U+E0D0-U+E0D2,U+E0D4,U+E700-U+E7C5,U+F000-U+F2E0,U+2665,U+26A1,U+F400-U+F4A8,U+F67C,U+E000-U+E00A,U+F300-U+F313,U+E5FA-U+E62B Symbols-1000-em Nerd Font Complete

enabled_layouts stack,tall
update_check_interval 0

include hostname-${HOSTNAME}.conf

map ctrl+alt+left neighboring_window left
map ctrl+alt+up neighboring_window up
map ctrl+alt+right neighboring_window right
map ctrl+alt+down neighboring_window down
```

and my open-actions.conf

```
# Tail a log file (*.log) in a new OS Window and reduce its font size
protocol file
ext log
action launch --title ${FILE} --type=os-window tail -f ${FILE_PATH}
action change_font_size current -2

# Open any file with a fragment in the editor, fragments are generated
# by the hyperlink_grep kitten and nothing else so far.
protocol file
fragment_matches [0-9]+
action launch --type=overlay $EDITOR +$FRAGMENT $FILE_PATH

# Open text files without fragments in the editor
protocol file
mime text/*
action launch --type=overlay $EDITOR $FILE_PATH

# Open image files in an overlay window with icat
protocol file
mime image/*
action launch --type=overlay kitty +kitten icat --hold $FILE_PATH

protocol filelist
action send_text all ${FRAGMENT}
```

**Reply by aorith** [reply]
Loving kitty so far. I'm trying to migrate my workflow from tmux. Here's my config if someone is interested: [Kitty](https://github.com/aorith/dotfiles/tree/master/topics/kitty/src/kitty)

**Reply by karimknaebel** [reply]
> symbol_map U+23FB-U+23FE,U+2B58,U+E200-U+E2A9,U+E0A0-U+E0A3,U+E0B0-U+E0BF,U+E0C0-U+E0C8,U+E0CC-U+E0CF,U+E0D0-U+E0D2,U+E0D4,U+E700-U+E7C5,U+F000-U+F2E0,U+2665,U+26A1,U+F400-U+F4A8,U+F67C,U+E000-U+E00A,U+F300-U+F313,U+E5FA-U+E62B Symbols-1000-em Nerd Font Complete

Just wondering, why do you need the `symbol_map` for Nerd Font? Although I couldn't find any documentation for this, just having 3270 nerd font installed, kitty seems to automatically fallback to it for PUA characters.

**Comment by hedefalk** [comment]
Pretty new user coming from iterm via a short stop at alacritty.

I've posted a couple of noob questions as bug reports as I couldn't find any forum like this so this is a great welcome for me :)

**Comment by kovidgoyal** [comment]
On Fri, Apr 09, 2021 at 10:59:44AM -0700, Karim Abou Zeid wrote:
> Just wondering, why do you need the `symbol_map` for Nerd Font? Although I couldn't find any documentation for this, just having 3270 nerd font installed, kitty seems to automatically fallback to it for PUA characters.

You dont strictly need it. If there is a single font on your system with
those symbols, then kitty will use them automatically. However as I have
a lot of fonts installed for testing I like to be explicit about which
font those symbols come from.
